megamek/src/megamek/common/Dropship.java

```diff
@@ -750,6 +753,9 @@ public boolean canLandVertically() {
      */
     @Override
     public boolean canTakeOffVertically() {
-        return (isSpheroid() || game.getPlanetaryConditions().isVacuum()) && (getCurrentThrust() > 2);
+        PlanetaryConditions conditions = game.getPlanetaryConditions();
+        boolean spheroidOrLessThanThin = isSpheroid()
+                || conditions.getAtmosphere().isLighterThan(Atmosphere.THIN);
```

Comment:
Trace is lighter than thin but not vacuum. Is this intended?

Reply:
see above